src/ingestion/kafka-queue.ts

```diff
@@ -130,7 +132,7 @@ export class KafkaQueue implements Queue {
 
     private static buildConsumer(kafka: Kafka): Consumer {
         const consumer = kafka.consumer({
-            groupId: 'plugin-server',
+            groupId: 'clickhouse-ingestion',
```

Comment:
is this something that could/should go in a setting/config as well?

Reply:
Not really, it'd mess ingestion up to change the group ID after starting it for the first time, so this is an internal detail (not worth it to bloat the settings list IMO).